src/taco-dependency-installer/dependencyInstaller.ts

```diff
@@ -355,14 +372,14 @@ module TacoDependencyInstaller {
 
             return installerUtils.promptUser(resources.getString("YesExampleString"))
                 .then(function (answer: string): Q.Promise<any> {
-                    if (answer === resources.getString("YesString")) {
+                    if (answer.toLocaleLowerCase() === resources.getString("YesString")) {
```

Comment:
That looks overkill for just checking for a "y" or a "Y"... Also, I believe we are going to go away from single key input (that is, the user will need to press "enter" in order for the input to be recorded) for the telemetry question. That library does not allow that behavior for a y/n prompt, so we would need to use just "question" method, which means we would need to manually check the answer with a check similar to what I'm doing here. There's also the fact that this is not localizable (the library doesn't let you customize the "y" and the "n" keys to something like "o" and "n" in French). I don't know if we're ever going to localize TACO, but it's something to keep in mind.

All in all, I don't think we should be taking an extra dependency for such a simple check.